Recrutify/Recrutify.Host/Controllers/ProjectController.cs

```diff
@@ -1,14 +1,17 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Recrutify.DataAccess.Models;
 using Recrutify.Services.DTOs;
 using Recrutify.Services.Services.Abstract;
 
 namespace Recrutify.Host.Controllers
 {
     [Route("api/[controller]")]
     [ApiController]
+    [Authorize(Roles = nameof(Role.Admin))]
     public class ProjectController : ControllerBase
```

Comment:
Write allow anonymous Attribute under get primary skills method, cause it should be available for unauthorized users